app/MetadataResolver/MetadataResolver.php

```diff
@@ -14,8 +14,10 @@ class MetadataResolver implements Resolver
         '~komiflo\.com(/#!)?/comics/(\\d+)~' => KomifloResolver::class,
         '~www\.melonbooks\.co\.jp/detail/detail\.php~' => MelonbooksResolver::class,
         '~ec\.toranoana\.(jp|shop)/(tora|joshi)(_[rd]+)?/(ec|digi)/item/~' => ToranoanaResolver::class,
-        '~iwara\.tv/(videos|images)/.*~' => IwaraResolver::class,
+        '~iwara\.tv/videos/.*~' => IwaraResolver::class,
```

Comment:
develop→feature branch マージのミス？

```suggestion
        '~iwara\.tv/(videos|images)/.*~' => IwaraResolver::class,
```